lab-lindsay/app/component/gallery/gallery-item/gallery-item.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+module.exports = {
+  template: require('./gallery-item.html'),
+  controllerAs: 'galleryItemCtrl',
+  controller: ['$log', 'galleryService', function($log, galleryService) {
+    $log.debug('Gallery Item Controller');
+
+    this.showEditGallery = false;
+    this.$onInit = () => {
+      $log.debug('Delete Gallery Item')
+      this.deleteGallery = () => {
+        galleryService.deleteGallery(this.gallery._id, this.gallery)
+        .then(
+          ()
```

Comment:
what are the operations you want to perform in this method? think about the data flow, and what you want rendered to page, and where that information comes from :)